examples/sqlx_any/Cargo.toml

```diff
@@ -25,6 +25,7 @@ sea-query-binder = { path = "../../sea-query-binder", features = [
     "with-bigdecimal",
     "with-uuid",
     "with-time",
+    "postgres-array",
```

Comment:
I added this feature because another crate in the workspace enabled sea-query/postgres-array, and so this cannot build